.github/workflows/full-suite.yml

```diff
@@ -338,6 +348,15 @@ jobs:
         run: |
           bundle config path vendor/bundle
           NODE_ENV=test RAILS_ENV=test ./bin/webpack
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          # Try to install libffi7 directly
+          sudo apt-get install -y libffi7 || {
+            # If that fails, download the package from Ubuntu archives and install it
+            wget http://archive.ubuntu.com/ubuntu/pool/main/libf/libffi/libffi7_3.3-4_amd64.deb
+            sudo dpkg -i libffi7_3.3-4_amd64.deb
```

Comment:
this is risky because it doesn't rewsolve the dependencies, did we tested it ?

Reply:
The first commit raised errors that this dependency is missing, which is why I am installing `libffi7`. Ah, I see your concern about the changes to `ubuntu-latest`. What do you think should be our solution then?